### ISSUE TITLE
[Bug]: X/Y/Z plot doesn't support ControlNet Integrated

### ISSUE THREAD

**Issue by aaambrozewicz** [lllyasviel/stable-diffusion-webui-forge#839]
### Checklist

- [ ] The issue exists after disabling all extensions
- [ ] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [ ] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

![2024-07-10_20h57_55](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/58288198/5164298e-fe47-4d85-843c-bacabf93ff70)
There is no controlnet options on XYZ plot script and u cannot install controlnet as a addon because Forge is automatically turning it off even if integrated controlnet is turned off.

### Steps to reproduce the problem

x

### What should have happened?

x

### Sysinfo

x

### Console logs

```Shell
x
```

**Comment by Panchovix** [comment]
Just if someone wants to know, you can disable the check that auto disables default controlnet being not enabled, but probably won't work
always_disabled_extensions = [
    #'sd-webui-controlnet',]

**Comment by aaambrozewicz** [comment]
> Just if someone wants to know, you can disable the check that auto disables default controlnet being not enabled, but probably won't work always_disabled_extensions = [ #'sd-webui-controlnet',]

Yeah, I del 'sd-webui-controlnet' in file "modules_forge\config.py" and "controlnet addon" show up in UI, but as you mention it doesn't work :(
